modin/experimental/core/execution/native/implementations/hdk_on_native/__init__.py

```diff
@@ -12,3 +12,107 @@
 # governing permissions and limitations under the License.
 
 """Experimental Modin's functionality related to Native execution engine and optimized for HDK storage format."""
+
+import modin.pandas as pd
```

Comment:
I think the new content of this file should be moved in modin/experimental/sql/hdk/query.py.